tests/summ_ai_api/summ_ai_test.py

```diff
@@ -100,7 +353,14 @@ async def test_auto_translate_easy_german(
     # Enable SUMM.AI in the test region
     await enable_summ_api(region_slug)
     # Setup a dummy server to fake responses from the SUMM.AI API server (an instance of aiohttp.test_utils.RawTestServer)
-    fake_server = await aiohttp_raw_server(fake_summ_ai_server)
+    fake_server = await aiohttp_raw_server(
+        make_mock_summ_ai_server(
+            response_data={
+                "translated_text": "Hier ist Ihre Leichte Sprache Übersetzung",
+                "jobid": "9999",
+            }
+        )
+    )
```

Comment:
```suggestion
    # Setup a dummy server to fake responses from the SUMM.AI API server
    # (an instance of aiohttp.test_utils.TestServer)
    fake_server = await aiohttp_server(make_mock_summ_ai_server_rate_limited())
    # Avoid unnecessary long wait times during tests
    settings.SUMM_AI_RATE_LIMIT_COOLDOWN = 0.1
```
In the real integration test I'd suggest to use the rate limited fake server to really test this feature.
This requires
```
from .utils import make_mock_summ_ai_server_rate_limited
```

and passing the fixture `aiohttp_server` instead of `aiohttp_raw_server` to `test_auto_translate_easy_german`.